index.js

```diff
@@ -634,7 +657,7 @@ function httpViewCleanup(db) {
   });
 }
 
-var localViewCleanup = utils.sequentialize(mainQueue, function (db) {
+function localViewCleanup(db) {
```

Comment:
Oh wait, I already added a test to do that. OK, I'm gonna take your +1 and merge then. :)